### PR TITLE
chore: generated CHANGELOG for 3.1.2

### PR DESCRIPTION
**Pull request by randi274**
### What does this PR do?

Change log for 3.1.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 3.1.2 - July 23, 2021
+
+## Fixed
+
+- Port pr v4.0.0 ([PR #387](https://github.com/forcedotcom/source-deploy-retrieve/pull/387))
```
Comment (xyc) [inline CHANGELOG.md:5]: We should remove this as this isn't a change for the next release.

**Comment by xyc** [comment]
@shetzel Could you let @randi274 know what you want to get released this week for SDR?

**Comment by randi274** [comment]
Thanks @xyc! That was most of my confusion for SDR, since it looked like this wasn't a change we needed to release. @shetzel was there something else that I missed, or that hasn't been merged yet?

**Comment by randi274** [comment]
@shetzel quick nudge on if you had anything to include. If not, we'll plan to skip the release this week.

**Pull request closed without merging**